src/types/onyx/Policy.ts

```diff
@@ -186,6 +186,7 @@ type Connection<ConnectionData, ConnectionConfig> = {
 
 type Connections = {
     quickbooksOnline: Connection<QBOConnectionData, QBOConnectionConfig>;
+    xero: Connection<QBOConnectionData, QBOConnectionConfig>;
```

Comment:
We have to define new types for Xero for sure, but it can also be handled in my PR with Importing data from Xero. 
It should be ready in a few days